contents/docker/Docker.md

```diff
@@ -0,0 +1,165 @@
+
+### Why learn Docker?
+
+Docker is a powerful tool that is rapidly gaining popularity. These are some statistics for Docker:
+
+<img src="Docker_Statistics.png" alt="Docker Statistics" width="60%">
+
+<sub>_Figure 3. Usage of Docker_ [(source)](https://www.docker.com/company)</sub>
+
+Many leading companies (e.g. Spotify, Nginx, ElasticSearch etc.) are using Docker for their deployment!
+
+If the numbers aren't enough to convince you to get started on Docker, as compiled by [RedHat](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/7.0_release_notes/sect-red_hat_enterprise_linux-7.0_release_notes-linux_containers_with_docker_format-advantages_of_using_docker) here are some of the notable advantages that Docker is able to provide:
+
+- **Rapid application deployment** – containers include the minimal runtime requirements of the application, reducing their size and allowing them to be deployed quickly.
+- **Portability across machines** – an application and all its dependencies can be bundled into a single container that is independent from the host version of Linux kernel, platform distribution, or deployment model. This container can be transferred to another machine that runs Docker, and executed there without compatibility issues.
+- **Version control and component reuse** – you can track successive versions of a container, inspect differences, or roll-back to previous versions. Containers reuse components from the preceding layers, which makes them noticeably lightweight.
+- **Sharing** – you can use a remote repository to share your container with others.Red Hat provides a registry for this purpose, and it is also possible to configure your own private repository.
+- **Lightweight footprint and minimal overhead** – Docker images are typically very small, which facilitates rapid delivery and reduces the time to deploy new application containers.
+- **Simplified maintenance** – Docker reduces effort and risk of problems with application dependencies.
+
+<box type="danger">
+As you can read from above, Docker is undeniably a powerful tool that can alleviate your deployment troubles. However, be aware that Docker is not a one-size-fits-all solution to your problems. Docker also has its limitations e.g. refer to this <a href="https://www.channelfutures.com/open-source/when-not-to-use-docker-understanding-the-limitations-of-containers/">article</a>. Carefully consider your use case before turning to Docker.
+</box>
+
+Now that we have a good idea of how Docker works and a summary of the notable advantages it is able to provide, let us take a closer look at some of the unique features that Docker is able to provide.
+
+#### Benefit: Docker is lightweight 
```

Comment:
It seems that **Benefits** and **why using docker** is a bit overlapping, consider merging these two sections into one? 
I use to use this organization and prof suggested me to reorganize into something like:

1. Wha is docker
2. Why use docker 
3. How to get stared